tests/ValueObject/ExportConfigTest.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DbExtractorConfig\Tests\ValueObject;
+
+use Keboola\DbExtractorConfig\Configuration\ValueObject\ExportConfig;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\IncrementalFetchingConfig;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\InputTable;
+use Keboola\DbExtractorConfig\Exception\InvalidArgumentException;
+use Keboola\DbExtractorConfig\Configuration\ValueObject\ValueObject;
+use PHPUnit\Framework\Assert;
+
+class ExportConfigTest extends BaseValueObjectTest
```

Comment:
Nevím, jestli bych to takhle extra testoval - testuješ prakticky jen gettery/settery. Přijde mi, že je tady navíc hodně "levels of indirection" - vlastně vůbec není vidět ten samotný test - co to testuje. Jakože - skoro by ten BaseValueObjectTest měl mít vlastní testy, jak je složitý.

Reply:
No ono to bolo povodne napisane pre testy ohladom metadat, a tam by sa to inak nedalo otestovat (ak by sa pisal test pre kazdy getter/setter ... validne, nevalidne hodnoty, tak by to bolo strasne vela riadkov). 

Mozem tam pridat aj nejake "klasicke" testy, .... toto testuje prakticky vsetky moznosti vramci rozsahu, ktory tam je zadany, co mi pride super ...

Reply:
Tohle je spíš pocit, no, sorry. Mě přijde, že vlastně nedokážu říct, co všechno to testuje a jestli to testuje správně. A to je pro mě 🟥 flag pro to, že je ten test moc chytrý :) Ale klidně to nech takto.